resources/views/PageDesign/PageDesign.twig

```diff
@@ -220,7 +220,7 @@
         "isCheapestSorting": "{{ services.template.isCheapestSorting() }}",
         "useVariationOrderProperties": {% if webstoreConfig.useVariationOrderProperties | default(0) %}true{% else %}false{% endif %},
         "initialData": {
-            {% if assetName == 'ceres-checkout' and services.country.hasEUShippingCountry() %}
+            {% if (assetName == 'ceres-checkout' or urls.templateType == 'register') and services.country.hasEUShippingCountry() %}
```

Comment:
Unfortunately we can't check for if the URL is register, the URL for this page can be different for each language. So it might work for /register but not for /en/register or /registrieren. We need to get the correct url or category id that is linked to the register content